herddb-core/src/main/java/herddb/model/planner/SemiJoinOp.java

```diff
@@ -82,6 +82,10 @@ public StatementExecutionResult execute(
         );
         EnumerableDataScanner joinedScanner = new EnumerableDataScanner(resRight.dataScanner.getTransaction(), fieldNames, columns, result, resLeft.dataScanner, resRight.dataScanner);
         return new ScanResult(resTransactionId, joinedScanner);
+    }
 
+    @Override
+    public String toString() {
+        return String.format("SemiJoinOp {leftKeySize = %d rightKeySize = %d }", leftKeys.length , rightKeys.length);
```

Comment:
Shall we tell more about the inputs? Like calling testing on them?

Reply:
@eolivelli Can you please explain what do you mean by testing?